src/rdkafka_conf.c

```diff
@@ -843,6 +843,14 @@ static const struct rd_kafka_property rd_kafka_properties[] = {
           _RK(group_id_str),
           "Client group id string. All clients sharing the same group.id "
           "belong to the same group." },
+        { _RK_GLOBAL|_RK_CGRP|_RK_MED, "group.instance.id", _RK_C_STR,
+          _RK(group_instance_id),
+          "Enables static group membership. "
```

Comment:
"Enable", to be consistent with existing properties

Reply:
This is not resolved

Reply:
Sort of , I took the liberty of adding a `s` when changing this to use `Enable`as it sounded better to me. Fixed to your exact specification this time.